_posts/2015-11-03-homogeneity.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Homogeneity, Main ennemy for DevOps?"
+reveal_theme: serif.css
+reveal_transition: linear
+---
+
+# Homogeneity, Main ennemy for DevOps?
+
+
+<small>Paris DevOps meetup 2015-11-03</small>
+
+--
+
+> Any intelligent fool can make things bigger and more complex...
+> It takes a touch of genius - and a lot of courage to move in the opposite direction.
+
+E. F. Schumacher
+
+---
+
+## Me
+
+- (Dev?)Ops in [www.themecloud.io](https://www.themecloud.io)
+- Worked for [Cloudwatt](https://www.cloudwatt.com) a public cloud privider
+  - And that's what we are talking about
+
+---
+
+## Cloudwatt
+
+<p class="stretch">
+![what we do]({{ site.baseurl }}/assets/img/homogeneity/whatwedo.png)
+</p>
+
+--
+
+### History
+
+- Silos + external providers + team architect
+- Silos + external providers
+- DevOps attempt
+
+---
+
+## Technology overview
+
+- KVM guest by service
+- Deploy by chef
+
+--
+
+- Chef as dumb android
+  - Reproduce commands / arch as you may have done manually
+- Automation to solve all problems at once
+
+--
+
+### Examples
+
+<p class="stretch">
+![net_sto]({{ site.baseurl }}/assets/img/homogeneity/net-sto.png)
+</p>
+
+--
+
+- Chef as Service Discovery
+
+---
+
+## Where is Problem?
+
+--
+
+* Choices never challenged
+  * Believe of unchangeable paragdigms
+    * Creation of cross dependencies
+      * Too hard to challenge choices
+        * Act like paradigms are accepted
+
+--
+
+<p class="stretch">
+![Pastafarianism]({{ site.baseurl }}/assets/img/homogeneity/pastafarianisme.jpg)
+</p>
+
+---
+
+## Evolution
+
+- Errors on duplication
+- Competition
+
+--
+
+<p class="stretch">
+![evolution]({{ site.baseurl }}/assets/img/homogeneity/evolution.gif)
+</p>
+
+--
+
+- Complexity manage by number of possibility
+- Problems lead to differentiation
+
+---
+
+## And for DevOps?
+
+--
+
+Don't fear new way of thinking
```

Comment:
Don't fear a new way of thinking / Don't fear new ways of thinking
L'un ou l'autre, selon ce qui te convient